src/telegram.ts

```diff
@@ -16,5 +16,12 @@ if (telegramBot) telegramBot.launch();
 
 export function sendTelegramMessage(message: string) {
   if (!telegramBot) return;
-  telegramBot.telegram.sendMessage(process.env.TELEGRAM_CHAT_ID || "", message);
+
+  // Multiple ChatIds have to be seperated by comma
+  // Example: TELEGRAM_CHAT_ID=xxxxxxxx, yyyyyyyy
+  var chatString = process.env.TELEGRAM_CHAT_ID || "";
+  var chats = chatString.split(",");
+  for ( var chatid of chats ) {
```

Comment:
Could you use `const` instead of `var` here since we are using Typescript?

Reply:
Ok, I've changed the declaration of the variables